source/guides/views/templates.md

```diff
@@ -19,6 +19,29 @@ The first is relative to the **_format_** and the latter is for the **_template
 For a given view named <code>Web::Views::Dashboard::Index</code>, there must be at least one template <code>dashboard/index.[format].[engine]</code> under the templates directory.
 </p>
 
+## Nested Templates
```

Comment:
Plus I would not write it with a subject like 'you'.

And I think there should be 2 sections instead of one. One for partial, and one for template